python/lsst/ts/ddsconfig/utils.py

```diff
@@ -18,7 +18,7 @@
 #
 # You should have received a copy of the GNU General Public License
 
-__all__ = ["get_pkg_root", "get_config_dir", "get_qos_path"]
+__all__ = ["get_pkg_root", "get_data_dir", "get_config_dir", "get_qos_path"]
```

Comment:
Please consider deleting get_pkg_root because any code using that is probably going to break with the new layout. The breakage may be easier to understand if the call fails, instead of returning a value that the caller can't properly use. (I also searched our code and found no callers, other than a unit test in this package.)

Reply:
Thanks! I wasn't sure if this function is used anywhere outside of ts_ddsconfig, which is why I left it in. I agree that the breakage may be easier to understand it the call fails so I have removed it.